dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -87,11 +88,20 @@ get_tools() {
         EXTRA_ARGS="-C $TOOLS_DIR"
     fi
 
-    if [ ! -f $OC_BINARY ]; then
-        echo "download oc client..."
+    download_oc() {
+        echo "download oc client $OC_VERSION"
         wget -q -O $TOOLS_DIR/$OC_PACKAGE $OC_BINARY_DOWNLOAD_URL
         eval "$ARCH" "$TOOLS_DIR"/"$OC_PACKAGE" "$EXTRA_ARGS" &>/dev/null
         rm -rf "$TOOLS_DIR"/README.md "$TOOLS_DIR"/LICENSE "${TOOLS_DIR:-/tmp}"/"$OC_PACKAGE"
+    }
+
+    if [[ ! -f $OC_BINARY ]]; then
+        download_oc
+    else
+        if [[ $($OC_BINARY version 2> /dev/null | grep "oc v" | cut -d " " -f2 | cut -d '+' -f1 || true) != *"$OC_VERSION"* ]]; then
```

Comment:
Can you add a comment to the file that describes what this check does and what input is expected

Reply:
sure